modules/react/badge/stories/examples/Basic.tsx

```diff
@@ -4,16 +4,19 @@ import {createStyles} from '@workday/canvas-kit-styling';
 import {base, system} from '@workday/canvas-tokens-web';
 
 const containerStyles = createStyles({
+  boxSizing: 'border-box',
```

Comment:
Just adding these for consistency.

Reply:
Should we add `boxSizing` to `createStyles()` as well?

Reply:
I had the same question for Nicholas. He said it's okay to add it in v11. I will update it 👍